src/StructuredLogViewer/Entrypoint.cs

```diff
@@ -10,6 +11,7 @@ public class Entrypoint
         [STAThread]
         public static void Main(string[] args)
         {
+            Debugger.Launch();

```

Comment:
probably don't want this change

Reply:
We don't want every MSBuildStructuredLogViewer user to try to debug it?? How else do we form the next generation of MSBuild devs? /s